drivers/staging/media/rpivid/rpivid_video.c

```diff
@@ -299,11 +302,11 @@ static int rpivid_try_fmt_vid_cap(struct file *file, void *priv,
 	int i;
 
 	/* Reject format types we don't support */
-	if (f->type != V4L2_BUF_TYPE_VIDEO_CAPTURE)
+	if (f->type != V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE)
```

Comment:
This check should be totally redundant as the framework won't allow it.

Reply:
Removed